client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -43,13 +43,36 @@ function createQuestionEpic(action$, state$, { window }) {
     ofType(actionTypes.createQuestion),
     tap(() => {
       const state = state$.value;
-      const challengeFiles = challengeFilesSelector(state);
+      let challengeFiles = challengeFilesSelector(state);
       const {
         title: challengeTitle,
         superBlock,
         block,
         helpCategory
       } = challengeMetaSelector(state);
+      if (
+        challengeFiles?.some(file => file.editableRegionBoundaries.length > 0)
+      ) {
+        const editableRegionStrings = fileExtension => {
+          const startComment = fileExtension === 'html' ? '<!--' : '/*';
+          const endComment = fileExtension === 'html' ? '-->' : '*/';
+          return `\n${startComment} 'User Editable Region' ${endComment}\n`;
```

Comment:
```suggestion
          return `\n${startComment} User Editable Region ${endComment}\n`;
```

I'm nitpicking - but we don't need quotes in the comment it looks weird.